lib/pco_api/people/person.ex

```diff
@@ -20,4 +19,15 @@ defmodule PcoApi.People.Person do
   linked_association :phone_numbers
   linked_association :school
   linked_association :social_profiles
+
+  def list, do: get("people/v2/people")
+  def list(params) when is_list(params), do: get(params, "people/v2/people")
```

Comment:
I was actually thinking something similar as I was driving home from my daughter's ballet. That would make it one function defined per action. I like it.